lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -7869,11 +7857,7 @@ public String updateQuestion() {
 		// Create or update a task
 		String reference = "/lessonbuilder/item/" + item.getId();
 		String title = getPage(item.getPageId()).getTitle() + " - " + messageLocator.getMessage("simplepage.question-task-title");
-		if (this.questionCreateTask) {
-			createOrUpdateTask(reference, getCurrentSiteId(), title, null);
-		} else {
-			updateTask(reference, title, null);
-		}
+		createOrUpdateTask(reference, getCurrentSiteId(), title, null);
```

Comment:
Could we add a comment here?
// Dashboard task widget:  When a question item is added create a task item for the site members.